### ISSUE TITLE
Multiple statements in @where broken in 0.1.3

### ISSUE THREAD

**Issue by equinetic** [JuliaStats/DataFramesMeta.jl#69]
A method error occurs when attempting to use more than one filter statement in `@where` with the most recent release.

**Version info:**
- Julia 0.6.0
- DataFramesMeta 0.1.3

**Code to reproduce error:**
```julia
Pkg.add("DataFramesMeta", v"0.1.3")
using DataFrames, DataFramesMeta

df = DataFrame(name=["John", "Sally", "Kirk"], age=[23., 42., 59.], children=[3,2,2])

@where(df, :name .== "John", :age .== 23)
```

**Error log:**
```julia
MethodError: &(::DataArrays.DataArray{Bool,1}, ::DataArrays.DataArray{Bool,1}) is ambiguous. Candidates:
  &(a::DataArrays.DataArray{Bool,N} where N, b::Union{AbstractArray{Bool,N} where N, Bool}) in DataArrays at /home/juser/.julia/v0.6/DataArrays/src/operators.jl:390
  &(b::Union{AbstractArray{Bool,N} where N, Bool}, a::DataArrays.DataArray{Bool,N} where N) in DataArrays at /home/juser/.julia/v0.6/DataArrays/src/operators.jl:390
Possible fix, define
  &(::DataArrays.DataArray{Bool,N} where N, ::DataArrays.DataArray{Bool,N} where N)
```

**Comment by ellisvalentiner** [comment]
Is this actually an issue in DataArrays.jl?

**Comment by ararslan** [comment]
Yes, and it will be fixed by https://github.com/JuliaStats/DataArrays.jl/pull/267.

**Comment by equinetic** [comment]
DataArrays #267 was merged into v0.6.1 and has resolved this issue. Thanks for the feedback guys!
